docs/add-ins/outlook-on-send-addins.md

```diff
@@ -334,18 +498,18 @@ function checkBodyOnlyOnSendCallBack(asyncResult) {
 
 The following are the parameters for the **addAsync** method:
 
-- *NoSend* &ndash; A string that is a developer-specified key to reference a notification message. You can use it to modify this message later. The key can’t be longer than 32 characters. 
+- *NoSend* &ndash; A string that is a developer-specified key to reference a notification message. You can use it to modify this message later. The key can’t be longer than 32 characters.
```

Comment:
Suggest formatting all parameter names as code (e.g., `onSend`) instead of italics -- to prevent parameter names from being localized.